lesson06/assignment/index.html

```diff
@@ -1,15 +1,240 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 06 - Bootstrap 4</title>
-    </head>
-    <body>
-        <main class="main">
-          
-        </main>
-    </body>
-</html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/js/bootstrap.bundle.min.js" integrity="sha384-gtEjrD/SeCtmISkJkNUaaKMoLD0//ElJ19smozuHV6z3Iehds+3Ulb9Bn9Plx0x4" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/05aeafdbc2.js" crossorigin="anonymous"></script>
+    <script src="./contact-form.js"></script>
+    <title>Assignment 06 - Bootstrap 5</title>
+  </head>
+  <body>
+    <main class="fw-light">
+      <header>
+        <div class="text-center">
+          <img class="img-fluid w-50 pt-4 p-3" src="./images/cpsn-logo.png" alt="Changes Parent Support Network logo">
+        </div>
+        <div class="text-center bg-image" style="
+        background-image: url('./images/frustrated-mom-daughter.jpg');
+        height: 250px;">
+          <div class="mask h-100" style="background-color: rgba(0, 0, 0, 0.2);">
```

Comment:
Inline styles are not best practice. Always use CSS in a dedicated .css file, or for this assignment, find the right classes and elements to get Bootstrap to do your styling for you.